tests/acceptance/cli_test.py

```diff
@@ -1227,7 +1228,7 @@ def test_events_json(self):
         os.kill(events_proc.pid, signal.SIGINT)
         result = wait_on_process(events_proc, returncode=1)
         lines = [json.loads(line) for line in result.stdout.rstrip().split('\n')]
-        assert [e['action'] for e in lines] == ['create', 'start', 'create', 'start']
+        self.assertEqual(Counter(e['action'] for e in lines), {'create': 2, 'start': 2})
```

Comment:
We prefer `assert` over `self.assertEqual`. `pytest` takes care of formatting the error properly, and generally does a better job than unittest at formatting it, and displaying relevant information.